components/Buttons/tests/unit/ButtonsShapeThemerTests.m

```diff
@@ -56,4 +57,10 @@ - (void)testMDCButtonShapeThemer {
                         self.shapeScheme.smallSurfaceShape.bottomRightCorner);
 }
 
+- (void)testBackgroundColorAfterButtonTheming {
+  MDCButtonScheme *scheme = [[MDCButtonScheme alloc] init];
+  [MDCTextButtonThemer applyScheme:scheme toButton:self.button];
+  XCTAssertEqualObjects([UIColor clearColor], self.button.backgroundColor);
+}
+
```

Comment:
If I understand correctly, the original issue was getting the background color after having a custom shape active. Can we add a test to cover that scenario (if it's not already covered by existing test(s)).

Reply:
done.